### DISCUSSION TITLE
Can swc be used for running cucumber.js scenarios?

### DISCUSSION THREAD

**Discussion by shaahinmo** [swc-project/swc#8037] [Q&A] [answered]
I have a Node.js project and I write scenarios using cucumber.js. But they are extremely slow when I run them. Can I use swc to run my scenarios?

**Comment by chunhoong** [comment]
Correct me if i am wrong, I am expecting swc to be a drop in replacement to ts-node loader.

Unfortunately

With ts-node (working):
```bash
node --loader ts-node/esm ./node_modules/@cucumber/cucumber/bin/cucumber.js
```

With swc(not working 😭 )
```bash
node --loader @swc-node/register/esm ./node_modules/@cucumber/cucumber/bin/cucumber.js
```

Would be glad if someone can show us the right way. Thanks! 🙏

**Reply by kdy1** [reply, marked as answer]
https://github.com/swc-project/swc/issues/7435
